networkx/algorithms/operators/unary.py

```diff
@@ -47,6 +55,13 @@ def reverse(G, copy=True):
     H : directed graph
         The reversed G.
 
+    Examples
```

Comment:
While we're updating this docstring, we should add a `Raises` section to indicate the `NetworkXError` that is raised if a user passes in an undirected graph!

Reply:
Okay sir, thanks, I will do that.